runtimes/peregrine/src/lib.rs

```diff
@@ -590,6 +590,8 @@ parameter_types! {
 	pub const MaxTotalKeyAgreementKeys: u32 = 1000;
 	#[derive(Debug, Clone, PartialEq)]
 	pub const MaxEndpointUrlsCount: u32 = 3u32;
+	// 300 blocks * 12s/block = 3.600s = 1h
+	pub const MaxBlocksTxValidity: u64 = 300u64;
```

Comment:
S: Add to primitives and use type `BlockNumber` instead of `u64` as you defined `type MaxBlocksTxValidity: Get<BlockNumberOf<Self>>;`.

Reply:
https://github.com/KILTprotocol/mashnet-node/pull/257/commits/c471802786366c73e670f33afc75d8498d1a0758

Reply:
Great improvement!